test-data/pybind11_mypy_demo/stubgen/pybind11_mypy_demo/basics.pyi

```diff
@@ -47,6 +47,7 @@ class Point:
     def __init__(self) -> None: ...
     @overload
     def __init__(self, x: float, y: float) -> None: ...
+    def as_list(self) -> List[float]: ...
```

Comment:
Probably out of scope, but is there a reason we don't emit `list[float]` here?

Reply:
My understanding is that this is simply what pybind11 puts in its `__doc__` strings, and the stub generator just forwards it. And if I have spotted it correctly in the upcoming release of pybind11 it may switch to annotating with lower-cased types, but I could be wrong about that.

Reply:
Correct, this is pybind11 behavior.